job-shop-scheduling/README.rst

```diff
@@ -1,8 +1,91 @@
-Demo of Job Shop Scheduling
-===========================
+Job Shop Scheduling Demo
+========================
+A demo on how to optimally schedule jobs using a quantum computer.
 
-Further Information
--------------------
+Given a set of jobs and a finite number of machines, how should you schedule
+your jobs on those machines such that all your jobs are completed at the
+earliest possible time? This question is the Job Shop Scheduling Problem!
+
+Each of our jobs can be broken down into smaller machine-specific tasks. For
+example, the job of making pancakes can be broken down into several
+machine-specific tasks: mixing ingredients in a *mixer* and cooking the batter
+on the *stove*. There is an order to these tasks (ex. you can't bake the batter
+before you mix the ingredients) and there is a time associated with each task
+(ex. 5 minutes on the mixer, 2 minutes to cook on the stove). Now supposing
+that you have multiple jobs with only a set number of machines, how do we
+schedule our tasks onto those machines so that our jobs complete as early
+as possible?
+
+Here is a breakfast example with making pancakes and frying some eggs:
+::
+
+  # Interpret as
```

Comment:
This comment becomes clear only after looking at the whole. Perhaps something to the effect of generic formulation or format